backend/drm/atomic.c

```diff
@@ -220,6 +225,10 @@ static bool atomic_crtc_commit(struct wlr_drm_connector *conn,
 
 	struct atomic atom;
 	atomic_begin(&atom);
+	if (conn->props.broadcast_rgb) {
```

Comment:
Is this required on every commit? I'd assume that it is enough to set this on modeset.

Reply:
We always set all of our state on each commit. Simplifies the code and I don't think it's a real performance concern.

Reply:
Makes sense.